server/@types/bapv.d.ts

```diff
@@ -7,6 +7,12 @@ export type Booker = {
   prisoners: Prisoner[] // prisoners this booker can book for
 }
 
+// data that is built up during an add prisoner journey
+export type AddPrisonerJourney = {
+  supportedPrisonIds: string[]
+  selectedPrisonId?: string
```

Comment:
Storing all `supportedPrisonIds` so they can be used in request validation in the `POST` route - and obviously the `selectedPrisonId` once validated (not implemented yet).